### PR TITLE
Fix logging version

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

bump logging version

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by mdayican** [comment]
> this is certainly not ready for a review. there are lot's of integration tests depending on logs and those need to be reviewed. i started locally but had to abandon it
> 
> This is not as straight forward as one might believe ;)

most of the log verification tests were for notification and they are removed.
only one test failing should_do_nothing_when_failing_to_get_container_client() which was not failing in my local and seems not related to logs.

**Comment by doncem** [comment]
> only one test failing should_do_nothing_when_failing_to_get_container_client() which was not failing in my local and seems not related to logs.

that's mentioned in our pull requests channel on slack and i did create a PR to address it (the name is wrong too): #1211

**Comment by mdayican** [comment]
> > only one test failing should_do_nothing_when_failing_to_get_container_client() which was not failing in my local and seems not related to logs.
> 
> that's mentioned in our pull requests channel on slack and i did create a PR to address it (the name is wrong too): #1211

so there is no other failing test

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/UploadEnvelopeDocumentsTaskTest.java [hunk 1]
```diff
@@ -60,27 +52,6 @@ void should_do_nothing_when_envelope_is_not_yet_ready_to_be_processed() {
         verifyNoMoreInteractions(envelopeRepository);
     }
 
-    // will verify grouping by container and throwing different error
```
Comment (mdayican) [inline src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/UploadEnvelopeDocumentsTaskTest.java:63]: removed this test for now. fix is #1211

**Pull request merged**